### PR TITLE
rt commands always returns exit code 0

### PR DESCRIPTION
**Pull request by Postyy**

### CODE REVIEW

**Comment by Postyy** [comment]
https://github.com/jfrog/jfrog-cli/issues/470

File: artifactory_test.go [hunk 1]
```diff
@@ -297,7 +298,55 @@ func TestAqlFindingItemOnRoot(t *testing.T) {
 	isExistInArtifactory(tests.GetSingleFileCopy(), searchPath, t)
 	cleanArtifactoryTest()
 }
+func TestExitCodeFlow(t *testing.T) {
```
Comment (eyalbe4) [inline artifactory_test.go]: I suggest renaming this function to **TestExitCode**.

File: utils/cliutils/utils.go [hunk 2]
```diff
@@ -68,16 +69,25 @@ func ExitOnErr(err error) {
 	}
 }
 
-func FailNoOp(err error, success, failed int, failNoOp bool) error {
-	if exitCode := GetExitCode(err, success, failed, failNoOp); exitCode == ExitCodeFailNoOp {
-		return CliError{exitCode, "No files were affected."}
+func CheckForErrorResults(err error, success, failed int, failNoOp bool) error {
```
Comment (eyalbe4) [inline utils/cliutils/utils.go]: I suggest renaming this function to **GetCliError**.

File: artifactory_test.go (hunk 1 above)
Comment (Postyy) [reply artifactory_test.go]: Fixed.

File: utils/cliutils/utils.go (hunk 2 above)
Comment (Postyy) [reply utils/cliutils/utils.go]: Fixed.

**Pull request merged**